products/sle/main.pm

```diff
@@ -1027,6 +1027,39 @@ sub load_slenkins_tests {
     return 0;
 }
 
+sub load_hpc_cluster_tests {
+    # We need to boot *before* waiting for support server
+    # Because of TianoCore UEFI boot method
+    loadtest "boot/boot_to_desktop";
+
+    # Wait for support server to complete its initialization
+    # Inspired by HA test, but we can't reuse it as it is
+    loadtest "hpc/wait_support_server";
+
+    # Standard boot and configuration
+    # Tests are disabled just for the development
+    #loadtest "qa_automation/patch_and_reboot" if is_updates_tests;
+    #loadtest "console/consoletest_setup";
+    loadtest "console/hostname" unless is_bridged_networking;
```

Comment:
same with this line and one line below , I vote just drop them

Reply:
"console/hostname" is test which sets hostname form HOSTNAME variable and updates dns running on support-server. This is mandatory as we rely on hostnames and we don't want to configure hostname in each test.

All cluster nodes should have same time. Although we are running in vm and all openQA workers should have same time, it is better to make sure that our nodes are in sync. Anyway I  don't insist on this step.